index.js

```diff
@@ -228,10 +231,10 @@ exports.connect = function(config, intern, callback) {
     host = config.host;
   }
 
-  if(config.port === undefined) {
-    port = 9160;
-  } else {
-    port = config.port;
```

Comment:
Well, first, I removed this code, because the `port` variable was not used anywhere after those lines... And, second, the Cassandra driver does not have a port parameter as such. Instead you can specify the port together with the host, like `127.0.0.1:9042`.
So, it is in fact not exactly possible nor convenient to have one single port parameter.